### PR TITLE
OKTA-444663 Update Groups API with Group Admins do not see Everyone group detail

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a bullet point to this section of the Groups API: [https://developer.okta.com/docs/reference/api/groups/#list-groups-with-defaults](https://developer.okta.com/docs/reference/api/groups/#list-groups-with-defaults) to address
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-444663](https://oktainc.atlassian.net/browse/OKTA-444663)

### CODE REVIEW

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/reference/api/groups/index.md [hunk 1]
```diff
@@ -235,11 +233,12 @@ Enumerates all Groups in your organization
...
-* If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
-* If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
-* The maximum value for `limit` is 200 for most orgs.
-* Don't write code that depends on the default or maximum value, as it may change.
-* If you receive an HTTP 500 status code, you have more than likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
+- If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
+- If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
+- The maximum value for `limit` is 200 for most orgs.
+- Don't write code that depends on the default or maximum value, as it may change.
+- If you receive an HTTP 500 status code, you have more than likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/groups/index.md:240]:
```suggestion
- If you receive an HTTP 500 status code, you likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
```

File: packages/@okta/vuepress-site/docs/reference/api/groups/index.md [hunk 2]
```diff
@@ -235,11 +233,12 @@ Enumerates all Groups in your organization
...
-* If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
-* If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
-* The maximum value for `limit` is 200 for most orgs.
-* Don't write code that depends on the default or maximum value, as it may change.
-* If you receive an HTTP 500 status code, you have more than likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
+- If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
+- If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
+- The maximum value for `limit` is 200 for most orgs.
+- Don't write code that depends on the default or maximum value, as it may change.
+- If you receive an HTTP 500 status code, you likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
```
Comment (sindhuvenkatesh-okta) [inline packages/@okta/vuepress-site/docs/reference/api/groups/index.md:240]: @brianduffield-okta : Please reach out to UD team to spot check the accuracy and precision of above statements as it stands today. I know you didn't add them; while we are at it - @niravkumarpatel-okta fyi

File: packages/@okta/vuepress-site/docs/reference/api/groups/index.md [hunk 3]
```diff
@@ -235,11 +233,12 @@ Enumerates all Groups in your organization
...
+- The Okta default Everyone group isn't returned for users with a Group Admin role.
```
Comment (sindhuvenkatesh-okta) [inline packages/@okta/vuepress-site/docs/reference/api/groups/index.md:241]: looks good to me

**Review by nghadge-okta — APPROVED** [review]

**Pull request merged**